contrib/qubes/meson.build

```diff
@@ -0,0 +1,55 @@
+dom0_scr = [
+  'src/__init__.py',
+  'src/fwupd_receive_updates.py',
+  'src/qubes_fwupd_heads.py',
+  'src/qubes_fwupd_update.py'
+]
+
+dom0_test = [
+  'test/__init__.py',
+  'test/fwupd_logs.py',
+  'test/test_qubes_fwupd_heads.py',
+  'test/test_qubes_fwupdmgr.py',
+]
+
+dom0_test_logs = [
+  'test/logs/firmware.metainfo.xml',
+  'test/logs/get_devices.log',
+  'test/logs/get_updates.log',
+  'test/logs/help.log',
+]
+
+dom0_test_logs_meta = [
+  'test/logs/metainfo_name/firmware.metainfo.xml',
+  'test/logs/metainfo_version/firmware.metainfo.xml',
+]
+
+vm_scripts = [
+  'src/vms/fwupd_common_vm.py',
+  'src/vms/fwupd_download_updates.py',
+  'src/vms/fwupd_usbvm_validate.py',
+]
+
+install_data(dom0_scr,
+             install_dir : 'share/qubes-fwupd/src')
+
+install_data(dom0_test,
+             install_dir : 'share/qubes-fwupd/test')
+
+install_data(dom0_test_logs,
+             install_dir : 'share/qubes-fwupd/test/logs')
+
+install_data(vm_scripts,
+             install_dir : 'libexec/qubes-fwupd',
+             install_mode : 'rwxrwxr-x')
```

Comment:
i think the meson standard is two space tabs rather than aligning to the bracket.

Reply:
I fixed the style in this file.